src/Datagrid/SimplePager.php

```diff
@@ -121,4 +121,9 @@ public function getThreshold(): int
     {
         return $this->threshold;
     }
+
+    public function isDeterministic(): bool
+    {
+        return false;
```

Comment:
There is maybe something to compute, because if we are on the last page and there is no next page, we know the exact number of result and we can return true.

Reply:
Sounds legit. I'll return this dynamically depending on currently known information!